src/annotator/highlighter.js

```diff
@@ -150,19 +144,48 @@ function drawHighlightsAbovePdfCanvas(highlightEl) {
  * @typedef {HTMLElement & HighlightProps} HighlightElement
  */
 
+/**
+ * Return the complete text nodes inside `range`.
```

Comment:
"complete text nodes"... did you mean "complete set of text nodes" here or "the set of all complete text nodes"?

Reply:
Ah. I meant "Return all the text nodes which are wholly inside range". I've rewritten this comment.